product_standard_margin/i18n/fr.po

```diff
@@ -4,106 +4,96 @@
+"Ce calcul fonctionne que le prix de vente soit exprimé en HT ou TTC dans Odoo. "
+"Si aucun prix de vente n'a été saisi, le taux sera de 999,0 %."
 
 #. module: product_standard_margin
 #: model:ir.model.fields,help:product_standard_margin.field_product_product__standard_markup_rate
 #: model:ir.model.fields,help:product_standard_margin.field_product_template__standard_markup_rate
 msgid ""
 "Markup rate is [ Theorical Margin / cost price (Wo Tax) ] of the product "
-"form (not based on historical values).Take care of tax include and exclude.. "
-"If no cost price set, will display 999.0"
+"form (not based on historical values).Take care of tax include and exclude.."
+" If no cost price set, will display 999.0"
 msgstr ""
-"Taux de marge théorique [ (PV unitaire - Coût unitaire ) / Coût unitaire du "
-"formulaire Produit] (ne tient pas compte de la facturation effective). Ce "
-"taux tient compte des PV TTC (TVA incluse). Si aucun coût n'a été saisi, le "
-"taux sera de 999.0"
+"Taux de marque théorique [ (Prix de vente HT - Coût ) / Coût ] du "
+"formulaire Produit (ne tient pas compte de la facturation effective). "
+"Ce calcul fonctionne que le prix de vente soit exprimé en HT ou TTC dans Odoo. "
+"Si aucun coût n'a été saisi, le taux sera de 999,0 %."
 
 #. module: product_standard_margin
 #: model:ir.model,name:product_standard_margin.model_product_template
 msgid "Product"
-msgstr "Article"
+msgstr "Produit"
 
 #. module: product_standard_margin
 #: model:ir.model,name:product_standard_margin.model_product_product
 msgid "Product Variant"
-msgstr "Variantes de produit"
+msgstr "Variante de produit"
 
 #. module: product_standard_margin
 #: model:ir.model.fields,field_description:product_standard_margin.field_product_product__list_price_vat_excl
 #: model:ir.model.fields,field_description:product_standard_margin.field_product_template__list_price_vat_excl
 msgid "Sale Price VAT Excluded"
-msgstr "Prix de vente (HT)"
+msgstr "Prix de vente HT"
 
 #. module: product_standard_margin
 #: model:ir.model.fields,field_description:product_standard_margin.field_product_product__standard_margin
 #: model:ir.model.fields,field_description:product_standard_margin.field_product_template__standard_margin
+#: model_terms:ir.ui.view,arch_db:product_standard_margin.view_product_product_form
+#: model_terms:ir.ui.view,arch_db:product_standard_margin.view_product_product_form_easy
+#: model_terms:ir.ui.view,arch_db:product_standard_margin.view_product_template_form
 msgid "Theorical Margin"
-msgstr "Marge Théorique"
+msgstr "Marge théorique"
 
 #. module: product_standard_margin
 #: model:ir.model.fields,field_description:product_standard_margin.field_product_product__standard_margin_rate
 #: model:ir.model.fields,field_description:product_standard_margin.field_product_template__standard_margin_rate
 msgid "Theorical Margin (%)"
-msgstr "Taux de marque Théorique (%)"
+msgstr "Taux de marge théorique (%)"
```

Comment:
This change is wrong : 
Margin is "Taux de marque" 
Markup is "Taux de marge".

(very funny, in fact ;-))